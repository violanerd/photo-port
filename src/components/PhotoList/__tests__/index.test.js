import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PhotoList from '..';

const category = "portraits"
afterEach(cleanup);

describe('PhotoList component', () => {
  it('renders', () => {
    render(<PhotoList category={category} />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<PhotoList category={category} />);
    
    expect(asFragment()).toMatchSnapshot();
  });
})