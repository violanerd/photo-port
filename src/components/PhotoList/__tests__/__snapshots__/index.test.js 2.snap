// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoList component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
    >
      <img
        alt="Cat green eyes"
        class="img-thumbnail mx-1"
        src="0.jpg"
      />
      <img
        alt="Green parrot"
        class="img-thumbnail mx-1"
        src="1.jpg"
      />
      <img
        alt="Yellow macaw"
        class="img-thumbnail mx-1"
        src="2.jpg"
      />
      <img
        alt="Pug smile"
        class="img-thumbnail mx-1"
        src="3.jpg"
      />
    </div>
  </div>
</DocumentFragment>
`;
